themes/default/content/blog/pulumi-targets-dotnet-6/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Pulumi now targets .NET 6"
+authors: ["zaid-ajaj"]
+tags: ["dotnet", "csharp", "fsharp", "vb.net"]
+meta_desc: "Introducing a static-code analyzer for C# which provides instant feedback on common mistakes defining Pulumi resources"
+# meta_image: enhanced-static-code-analysis.png
+date: "2022-07-20"
+---
+
+In this blog post, we will talk about how Pulumi is now using [.NET 6](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-6) as our default across the ecosystem. We will discuss the changes applied to templates, program structure and code generation. We also explain how Pulumi C# projects can benefit from latest features in .NET 6 and how it simplifies your programs overall. Let's dive in, shall we?
+
+<!--more-->
+
+### Templates
+
+All of our .NET templates have been updated which now use `net6.0` as the target framework. This means that projects created with `pulumi new` require at least the [.NET 6 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) to build and run the projects. The templates also simplified the program structure quite a bit and not longer use `MyStack` classes to define resources in case of C# and VB.NET templates. F# templates didn't change that much since these didn't use the class approach to define infrastructure. 
```

Comment:
```suggestion
All of our .NET templates have been updated which now use `net6.0` as the target framework. This means that projects created with `pulumi new` require at least the [.NET 6 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) to build and run the projects. The templates also simplified the program structure quite a bit and not longer use `MyStack` classes to define resources in case of C# and VB.NET templates. F# templates didn't change that much since these already didn't use the class approach to define infrastructure. 
```